pkgs/development/python-modules/isort/default.nix

```
@@ -2,21 +2,19 @@
 
 buildPythonPackage rec {
   pname = "isort";
-  version = "4.3.4";
+  version = "4.3.15";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1y0yfv56cqyh9wyg7kxxv9y5wmfgcq18n7a49mp7xmzka2bhxi5r";
+    sha256 = "1dmxbq9cd45j27f7inmz306vd0a76iflk5hqk01nawvf5k51y5cn";
   };
 
   propagatedBuildInputs = lib.optional isPy27 futures;
 
   checkInputs = [ mock pytest ];
 
   checkPhase = ''
-    py.test test_isort.py -k "not test_long_line_comments \
-                          and not test_import_case_produces_inconsistent_results_issue_472 \
-                          and not test_no_extra_lines_issue_557"
+    TMPDIR=/tmp/ python3 -m pytest
```

```suggestion
    TMPDIR=/tmp/ pytest
```
or use `${python.interpreter}`.
Please also add a comment on why TMPDIR is set.

---

Done.